CHANGELOG.md

```diff
@@ -0,0 +1,268 @@
+    ```
+    BREAKING CHANGE:
+    requires SecretStore.Type be set to `vault`
+    ```
+    
+- Added new secrets configuration parameters to EncryptWithAES ([#193](https://github.com/edgexfoundry/app-service-configurable/issues/193)) ([#0f5331e](https://github.com/edgexfoundry/app-service-configurable/commits/0f5331e))
+
+- Update profiles for new filter enhancements ([#187](https://github.com/edgexfoundry/app-service-configurable/issues/187)) ([#8e9141f](https://github.com/edgexfoundry/app-service-configurable/commits/8e9141f))
+
+- Update profiles for new subscribe topic for V2 DTOs ([#175](https://github.com/edgexfoundry/app-service-configurable/issues/175)) ([#b2b5516](https://github.com/edgexfoundry/app-service-configurable/commits/b2b5516))
+
+- Update profiles for multiple MessageBus subscriptions ([#164](https://github.com/edgexfoundry/app-service-configurable/issues/164)) ([#2d8b7de](https://github.com/edgexfoundry/app-service-configurable/commits/2d8b7de))
+
+- Remove MarkAsPushed feature ([#159](https://github.com/edgexfoundry/app-service-configurable/issues/159)) ([#ed155f3](https://github.com/edgexfoundry/app-service-configurable/commits/ed155f3))
+
+    ```
+    BREAKING CHANGE:
+    MarkedAsPushed capability no longer supported
+    ```
+
+- Update configuration files to set default non-TLS ([#157](https://github.com/edgexfoundry/app-service-configurable/issues/157)) ([#0b6ac86](https://github.com/edgexfoundry/app-service-configurable/commits/0b6ac86))
+
+- Remove remote logging capability ([#143](https://github.com/edgexfoundry/app-service-configurable/issues/143)) ([#dc1b719](https://github.com/edgexfoundry/app-service-configurable/commits/dc1b719))
+### Bug Fixes 🐛
+- Make default http export secret settings blank as default ([#280](https://github.com/edgexfoundry/app-service-configurable/issues/280)) ([#712355d](https://github.com/edgexfoundry/app-service-configurable/commits/712355d))
+- Remove retry related properties and update secret path of SecretStore config ([#272](https://github.com/edgexfoundry/app-service-configurable/issues/272)) ([#abca950](https://github.com/edgexfoundry/app-service-configurable/commits/abca950))
+- Fix docker image version version built locally ([#251](https://github.com/edgexfoundry/app-service-configurable/issues/251)) ([#adb0691](https://github.com/edgexfoundry/app-service-configurable/commits/adb0691))
+- **snap:** fix README's security-secret-store doc ([#207](https://github.com/edgexfoundry/app-service-configurable/issues/207)) ([#a4c663a](https://github.com/edgexfoundry/app-service-configurable/commits/a4c663a))
+- **snap:** allow disabling of secret-store via config hook ([#151](https://github.com/edgexfoundry/app-service-configurable/issues/151)) ([#8bb6462](https://github.com/edgexfoundry/app-service-configurable/commits/8bb6462))
+### Code Refactoring ♻
```

Comment:
Please insert new lines before and after title- easier to read the separation boundary and also lint from markdown

Reply:
Renders fine w/o extra lines

Reply:
`lint from markdown` ???

Reply:
yea, i have that from VS code - part of extension or plugin.  so it will show some lint issue (like underline different color) right inside the editor.